solution-base/zookeeper/operator.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: zookeeper-operator
+  namespace: ${ZOOKEEPER_NAMESPACE}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: zookeeper-operator
+  template:
+    metadata:
+      labels:
+        name: zookeeper-operator
+    spec:
+      serviceAccountName: zookeeper-operator
+      containers:
+        - name: zookeeper-operator
+          # Replace this with the built image name
+          image: ${ZOOKEEPER_IMAGE_PREFIX}/zookeeper-operator:${ZOOKEEPER_OPERATOR_TAG}
```

Comment:
I saw that we used `KUBEDB_IMAGE_NAME` or `KAFKA_OPERATOR_IMAGE_NAME` (don't know if that was the exact name but the idea was there) should we make one for `zookeeper-operator` as well?

Reply:
done*